Sources/ImagePickerController.swift

```diff
@@ -56,8 +56,16 @@ open class ImagePickerController: UIImagePickerController, TypedRowControllerTyp
     
     open func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
         (row as? ImagePickerProtocol)?.imageURL = info[UIImagePickerController.InfoKey.referenceURL] as? URL
+
+        row.value = info[UIImagePickerController.InfoKey.originalImage] as? UIImage
```

Comment:
Setting the row value will trigger the `onChange` callback so it would be better to only assign a value once in this function, after checking whether to use the edited image.

Otherwise this change looks good

Reply:
Great observation, I also will only set the row.value on the end of the scope before completion, that way we don't risk triggering the callback with some data unavailable like the `userPickerInfo `